jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -101,6 +103,12 @@
               "python-heat-cfntools",
               "openstack-rally",
               "openstack-tempest",
+            ].contains(component)  ||
+            # Pike only
+            [ "Cloud:OpenStack:Master", "Cloud:OpenStack:Ocata:Staging", "Cloud:OpenStack:Newton:Staging" ].contains(project) &&
```

Comment:
so why are they not in Master?

Reply:
Because we maintain them upstream so the packages are in Cloud:OpenStack:Upstream:Master and will be updated there automatically.